mmeds/validate.py

```diff
@@ -324,8 +324,11 @@ def check_column(self, column):
             for i, cell in enumerate(column):
                 if pd.isna(cell):
                     # Check for missing required fields
+                    # Subject ID allowed to be NA if non-NA Subject ID of another type in the same row
+                    Logger.debug(f"complement: {complement}")
```

Comment:
I like passing in arguments with defaults of None but there should be a check that complement is not None before using it. Python here might not throw a None type error but other languages might not be so forgiving